### ISSUE TITLE
There is no accurate error hint for this error yet. We are continuously in the process of increasing hint for this kind of error check. It would be helpful if you could inform us of how this conversion went by opening a github issue. And we will resolve it with high priority.

### ISSUE THREAD

**Issue by Colorfu1** [PaddlePaddle/Paddle#22946]
- 版本、环境信息：
    1）PaddlePaddle版本：1.7
    2）CPU：Intel Xeon CPU E3-1240 v5
    3）GPU：无
    4）系统环境：Windows
- 复现信息：如果需要复现代码，可以提供
- 问题描述：使用Paddle内置LSTMCell模块。出现报错。
https://www.paddlepaddle.org.cn/documentation/docs/zh/1.6/api_cn/layers_cn/LSTMCell_cn.html#lstmcell
----------------------
Error Message Summary:
----------------------
Error: An error occurred here. There is no accurate error hint for this error yet. We are continuously in the process of increasing hint for this kind of error check. It would be helpful if you could inform us of how this conversion went by opening a github issue. And we will resolve it with high priority.
  - New issue link: https://github.com/PaddlePaddle/Paddle/issues/new
  - Recommended issue content: all error stack information
  [Hint: Expected desc->total_size >= size, but received desc->total_size:0 < size:4198400.] at (D:\1.7.0\paddle\paddle\fluid\memory\detail\memory_block.cc:41)
  [operator < concat > error]
I0311 11:19:17.839036 21908 parallel_executor.cc:440] The Program will be executed on CPU using ParallelExecutor, 1 cards are used, so 1 programs are executed in parallel.
I0311 11:19:17.859036 21908 build_strategy.cc:365] SeqOnlyAllReduceOps:0, num_trainers:1
I0311 11:19:17.927037 21908 parallel_executor.cc:307] Inplace strategy is enabled, when build_strategy.enable_inplace = True
I0311 11:19:18.041036 21908 parallel_executor.cc:322] Cross op memory reuse strategy is enabled, when build_strategy.memory_optimize = True or garbage collection strategy is disabled, which is not recommended

出错的代码块应该是：
![捕获](https://user-images.githubusercontent.com/30551427/76380353-828c6900-638d-11ea-9cde-d8482e7b8943.PNG)
lstm的list里面是3个LSTMCell，输入为（B, seq_len, C, H, W）,每次从输入里取一个（B,  C, H, W），经过3个LSTMCell得到结果，把结果存起来。

**Comment by 1024er** [comment]
能在hi上联系@wuxing 提供一下复现环境么？

**Comment by 1024er** [comment]
> * 版本、环境信息：
>      1）PaddlePaddle版本：1.7
>      2）CPU：Intel Xeon CPU E3-1240 v5
>      3）GPU：无
>      4）系统环境：Windows
> * 复现信息：如果需要复现代码，可以提供
> * 问题描述：使用Paddle内置LSTMCell模块。出现报错。
>   https://www.paddlepaddle.org.cn/documentation/docs/zh/1.6/api_cn/layers_cn/LSTMCell_cn.html#lstmcell
> 
> ## Error Message Summary:
> Error: An error occurred here. There is no accurate error hint for this error yet. We are continuously in the process of increasing hint for this kind of error check. It would be helpful if you could inform us of how this conversion went by opening a github issue. And we will resolve it with high priority.
> 
> * New issue link: https://github.com/PaddlePaddle/Paddle/issues/new
> * Recommended issue content: all error stack information
>   [Hint: Expected desc->total_size >= size, but received desc->total_size:0 < size:4198400.] at (D:\1.7.0\paddle\paddle\fluid\memory\detail\memory_block.cc:41)
>   [operator < concat > error]
>   I0311 11:19:17.839036 21908 parallel_executor.cc:440] The Program will be executed on CPU using ParallelExecutor, 1 cards are used, so 1 programs are executed in parallel.
>   I0311 11:19:17.859036 21908 build_strategy.cc:365] SeqOnlyAllReduceOps:0, num_trainers:1
>   I0311 11:19:17.927037 21908 parallel_executor.cc:307] Inplace strategy is enabled, when build_strategy.enable_inplace = True
>   I0311 11:19:18.041036 21908 parallel_executor.cc:322] Cross op memory reuse strategy is enabled, when build_strategy.memory_optimize = True or garbage collection strategy is disabled, which is not recommended
> 
> 出错的代码块应该是：
> ![捕获](https://user-images.githubusercontent.com/30551427/76380353-828c6900-638d-11ea-9cde-d8482e7b8943.PNG)
> lstm的list里面是3个LSTMCell，输入为（B, seq_len, C, H, W）,每次从输入里取一个（B, C, H, W），经过3个LSTMCell得到结果，把结果存起来。

线下已经沟通。如果没有更多的问题，请将卡片close吧。
